.gitignore

```
@@ -12,4 +13,18 @@ node_modules
 yarn-error.log
 # DS _store
 .DS_Store
-.vscode
\ No newline at end of file
+
+# unnecessary terraform files
+*.tfstate
+*.tfstate.backup
+*.tfstate.lock.info
+
+# logs
+*.log
+
+# Directories
+.terraform/
+
+# lambda
+**/cronMailing/*
+!**/cronMailing/*.py
```

can you help me add a new line here, assuming eslint doesn't complain
```suggestion
!**/cronMailing/*.py

```